public/stylesheets/styles.css

```diff
@@ -31,15 +33,79 @@ h4 {
 }
 
 .footer {
-  position: relative;
+  position: fixed;
   left: 0;
   bottom: 0;
+  z-index: 999;
```

Comment:
I changed this back to fixed, I think moving it to relative made the footer too close to the page's content. I checked the pages and the only place I saw overlap with the footer was when the user tried to remove the movie/TV show from their watch list. When that modal pops up, it overlaps with the footer. We can try to set the modal's z-index so that it doesn't overlap, but I think that's a small area of the site that we can prioritize later, or we can wait for Cheng to merge his fix in and see if it fixes it?